### PR TITLE
Chain link

### PR DESCRIPTION
**Pull request by bmatherly**
Add Chain/Link documentation
Move MLT++ documentation from git

### CODE REVIEW

File: developers/mlt++.md [hunk 1]
```diff
@@ -0,0 +1,566 @@
...
+
+~~~
+#include <mlt++/Mlt.h>
+using namespace Mlt;
+
+int main( void )
+{
+    Factory::init( );
+    Producer p( "pango:", "Hello World" );
+    Consumer c( "sdl" );
```
Comment (ddennedy) [inline developers/mlt++.md]: Where did this document come from? This example is no longer correct.

File: developers/mlt++.md [hunk 2]
```diff
@@ -0,0 +1,566 @@
...
+{
+    Factory::init( );
+    Playlist list;
+    for ( int i = 1; i < argc; i ++ )
+    {
+        Producer p( argv[i] );
+        if ( p.is_valid( ) )
+            list.append( p );
+    }
+    Consumer c( "sdl" );
```
Comment (ddennedy) [inline developers/mlt++.md]: This one too. Missing a profile. And others to follow.

File: developers/mlt++.md (hunk 1 above)
Comment (bmatherly) [reply developers/mlt++.md]: I migrated this file - which I think we could remove now:
https://github.com/mltframework/mlt/blob/master/docs/mlt%2B%2B.txt

Fixed

File: developers/mlt++.md (hunk 2 above)
Comment (bmatherly) [reply developers/mlt++.md]: Fixed

**Comment by ddennedy** [comment]
THANK YOU. I will fix some of the other code in Framework.md. I previously fixed the first main one that people were copying and pasting but overlooked the others.

**Comment by bmatherly** [comment]
Is it OK if I remove this file?
https://github.com/mltframework/mlt/blob/master/docs/mlt%2B%2B.txt

**Comment by ddennedy** [comment]
yes

**Pull request merged**